src/App.js

```diff
@@ -0,0 +1,21 @@
+// import logo from './logo.svg';
+import './App.css';
+
+import React from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import Greeting from './components/Greeting';
+
+function App() {
+  return (
+    <Router>
+      <div className="App">
+        <Routes>
+          <Route path="/" element={<Greeting />} />
+          {/* Add more routes here */}
+        </Routes>
```

Comment:
- [ ] Dear @massazoum, Your backend is setup correctly and returns the greeting message well. But, running the project locally, I'm facing an issue with the frontend `has been blocked by CORS policy: ` and the application is failing to featch the greeting message` Please take a look into the error and address the issue. 

### The app 

https://github.com/massazoum/hello-react-front-end/assets/69077061/1b6d1311-ec2c-43f7-a23c-92dfeed4d41f